src/enrichment_facility.h

```diff
@@ -72,70 +79,90 @@ class NatUConverter : public cyclus::Converter<cyclus::Material> {
+///  to -1 and the material is not accepted.  Any material components other
+///  other than U235 and U238 are sent directly to the tails buffer.
+
+///  EnrichmentFacility will bid on any request for its output commodity
+///  up to the maximum allowed enrichment (if not specified, default is 100%)
+///  It bids on either the request quantity, or the maximum quanity allowed
+///  by its SWU constraint or natural uranium inventory, whichever is lower.
+///  If multiple output commodities with different enrichment levels are
+///  requested and the facility does not have the SWU or quantity capacity
+///  to meet all requests, the requests are fully, then partially filled
+///  in unspecified but repeatable order.
+
+///  EnrichmentFacility also offers its tails as an output commodity with
+///  no associated recipe.  Bids for tails are constrained only by total
+///  tails inventory.
+
 class EnrichmentFacility : public cyclus::Facility {
+#pragma cyclus note {   	  \
+  "niche": "enrichment facility",				  \
+  "doc":								\
+  "EnrichmentFacility is a simple Agent that enriches natural"		\
+  "uranium in a Cyclus simulation. It does not explicitly compute"	\
+  "the physical enrichment process, rather it calculates the SWU"	\
+  "required to convert an source uranium recipe (ie. natural uranium)"	\
+  "into a requested enriched recipe (ie. 4% enriched uranium), given"	\
+  "the natural uranium inventory constraint and its SWU capacity"	\
+  "constraint."								\
+  "\n\n"								\
+  "Enrichment Facility requests an input commodity and associated recipe" \
+  "whose quantity is its remaining inventory capacity.  All facilities" \
+  "trading the same input commodity (even with different recipes) will" \
+  "offer materials for trade.  The Enrichment Facility accepts any input" \
+  "materials with enrichments less than its tails assay, as long as some" \
+  "U235 is present, and preference increases with U235 content.  If no" \
+  "U235 is present in the offered material, the trade preference is set" \
+  "to -1 and the material is not accepted.  Any material components other" \
+  "other than U235 and U238 are sent directly to the tails buffer."	\
+  "\n\n"								\
+  "EnrichmentFacility will bid on any request for its output commodity" \
+  "up to the maximum allowed enrichment (if not specified, default is 100%)" \
+  "It bids on either the request quantity, or the maximum quanity allowed" \
+  "by its SWU constraint or natural uranium inventory, whichever is lower." \
+  "If multiple output commodities with different enrichment levels are" \
+  "requested and the facility does not have the SWU or quantity capacity" \
+  "to meet all requests, the requests are fully, then partially filled" \
+  "in unspecified but repeatable order."				\
+  "\n\n"								\
+  "EnrichmentFacility also offers its tails as an output commodity with" \
+  "no associated recipe.  Bids for tails are constrained only by total" \
```

Comment:
"Recipe" used in this context is a bit strange.  The offerred tails material has a composition - and we offer the actual composition that we will be trading away.  So I think something like:

```
Accumulated tails inventory is offered for trading on a specifiable output commodity.
```

No need to mention constraints - that is implicit - the enrichment facility can't ever trade away more tails than it has :-)